### ISSUE TITLE
Cannot read property 'loadFile' of null

### ISSUE THREAD

**Issue by AndrewMorsillo** [BetterThanTomorrow/calva#131]
This error toast pops up on vscode 1.31.1 and Calva 1.3.63 on MacOS 10.14.1 when attempting to evaluate any cljs file.

**Comment by PEZ** [comment]
Hello,

I am sorry for the inconvenience! Sounds like we need better diagnostics and error messages here. Somehow the Calva REPL client fails to initialize for you.

What kind of project is this? Figwheel? Shadow-CLJS? Something else? Do you get any error messages in the terminal where you have started the REPLs?

Also, please feel invited to the [#calva-dev](https://clojurians.slack.com/?redir=%2Fmessages%2Fcalva-dev%2F) channel of the Clojurians slack, where there are more people who can try figure this out and get your cljs files working in Calva.

**Comment by AndrewMorsillo** [comment]
It is a Leiningen Figwheel project. No errors or anything in the repl where the project is running. Project runs just fine. Here is the project.clj if that will help. The project is just the bare template provided by "lein new expo" with cider added. I'm new to clj/cljs so I may have misconfigured something.

```
; run lein repl then this
; (require '[figwheel-sidecar.repl-api :as figwheel]) (figwheel/start-figwheel!) (figwheel/cljs-repl)
; to connect from vscode calva
(defproject test-native-cljs "0.1.0-SNAPSHOT"
  :description "FIXME: write description"
  :url "http://example.com/FIXME"
  :license {:name "Eclipse Public License"
            :url  "http://www.eclipse.org/legal/epl-v10.html"}
  :dependencies [[org.clojure/clojure "1.9.0"]
                 [org.clojure/clojurescript "1.10.238"]
                 [javax.xml.bind/jaxb-api "2.4.0-b180830.0359"]
                 [binaryage/oops "0.5.8"]
                 [reagent "0.7.0" :exclusions [cljsjs/react cljsjs/react-dom cljsjs/react-dom-server cljsjs/create-react-class]]
                 [re-frame "0.9.3"]
                 [react-native-externs "0.1.0"]]
  :plugins [[lein-cljsbuild "1.1.7" :exclusions [[org.clojure/clojure]]]
            [lein-figwheel "0.5.16"]]
  :clean-targets ["target/" "main.js"]
  :aliases {"figwheel"        ["run" "-m" "user" "--figwheel"]
                      ; TODO: Remove custom extern inference as it's unreliable
                      ;"externs"         ["do" "clean"
                      ;                   ["run" "-m" "externs"]]
            "rebuild-modules" ["run" "-m" "user" "--rebuild-modules"]
            "prod-build"      ^{:doc "Recompile code with prod profile."}
            ["with-profile" "prod" "cljsbuild" "once" "main"]}
  :profiles {:dev  {:plugins [[cider/cider-nrepl "0.21.1"]]
                    :dependencies [[figwheel-sidecar "0.5.16"]
                                   [cider/piggieback "0.4.0"]]
                    :source-paths ["src" "env/dev"]
                    :cljsbuild    {:builds [{:id           "main"
                                             :source-paths ["src" "env/dev"]
                                             :figwheel     true
                                             :compiler     {:output-to     "target/expo/not-used.js"
                                                            :main          "env.expo.main"
                                                            :output-dir    "target/expo"
                                                            :optimizations :none}}]}
                    :repl-options {:nrepl-middleware [cider.piggieback/wrap-cljs-repl]}}
             :prod {:cljsbuild {:builds [{:id           "main"
                                          :source-paths ["src" "env/prod"]
                                          :compiler     {:output-to          "main.js"
                                                         :main               "env.expo.main"
                                                         :output-dir         "target/expo"
                                                         :static-fns         true
                                                         :externs            ["js/externs.js"]
                                                         :infer-externs      true
                                                         :parallel-build     true
                                                         :optimize-constants true
                                                         :optimizations      :advanced
                                                         :closure-defines    {"goog.DEBUG" false}}}]}}})

```

**Comment by PEZ** [comment]
I wonder if it could be that Calva can't handle that modern version of cider-nrepl...

Can you try starting the REPL with this command:

```sh
/usr/local/bin/lein update-in :dependencies conj "[nrepl \"0.5.3\"]" -- update-in :dependencies conj "[cider/piggieback \"0.3.10\"]" -- update-in :plugins conj "[cider/cider-nrepl \"0.20.0\"]" -- update-in "[:repl-options :nrepl-middleware]" conj "[\"cider.nrepl/cider-middleware\"]" -- update-in "[:repl-options :nrepl-middleware]" conj "[\"cider.piggieback/wrap-cljs-repl\"]" -- repl :headless
```

**Comment by baflo** [comment]
Hi, I'm an absolute beginner with Clojure/Clojurescript coming from the JavaScript world... that's why I start with ClojureScript. I have the same error as @AndrewMorsillo with VScode Insiders 1.36.0 and VSCode 1.35.0 on Windows. This is my current `project.clj`:

```clojure
(defproject lein-cljsbuild-bootstrap "0.0.1"
  :plugins [[lein-cljsbuild "1.1.7"]]
  :dependencies [[org.clojure/clojure "1.10.1"]
                 [org.clojure/clojurescript "1.10.520"]]
  :cljsbuild {:builds [{; The path to the top-level ClojureScript source directory:
                        :source-paths ["src-cljs"]
                        ; The standard ClojureScript compiler options:
                        ; (See the ClojureScript compiler documentation for details.)
                        :compiler {:output-to "main.js"  ; default: target/cljsbuild-main.js
                                   :output-dir "target"
                                   :optimizations :whitespace
                                   :pretty-print true}}]})
```

**Comment by PEZ** [comment]
@baflo , sorry for late reply. Do you get this still? And do you get it even if you use Jack-in?

**Comment by cfehse** [comment]
I am closing this issue for now. This should be fixed in version 2.0.55.
